### PR TITLE
Add more documentation about cluster.restart()

### PR DESCRIPTION
**Pull request by BelSasha**

### CODE REVIEW

**Comment by BelSasha** [comment]
* **#825** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/825?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @BelSasha and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: runhouse/servers/http/http_server.py [hunk 1]
```diff
@@ -1023,6 +1015,10 @@ async def main():
         "den_auth", False
     )
     cluster_config["den_auth"] = den_auth
+    if den_auth:
+        cluster_config["den_status_ping_interval"] = DEFAULT_STATUS_CHECK_INTERVAL
```
Comment (rohinb2) [inline runhouse/servers/http/http_server.py:1019]: why do we still set this in the cluster_config?

Comment (rohinb2) [reply runhouse/servers/http/http_server.py:1019]: shouldn't this just not be set at all, and use den_auth

**Comment by rohinb2** [comment]
### Merge activity

* **May 23, 1:50 PM EDT**: @rohinb2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/run-house/runhouse/825).

**Pull request merged**